cmd/main.go

```diff
@@ -102,6 +107,14 @@ func init() {
 		os.Exit(0)
 	}
 
+	// Set some configuration defaults
+	ko.Load(structs.Provider(
+		metrics.PrometheusConfig{
```

Comment:
I think the correct way here is to use `ko.Unmarshal`, like done here: https://github.com/knadh/listmonk/blob/master/cmd/init.go#L239-L241